sdk/provisioning/Generator/src/Generator.csproj

```diff
@@ -38,12 +38,14 @@
     <PackageReference Include="Azure.ResourceManager.OperationalInsights" />
     <PackageReference Include="Azure.ResourceManager.PostgreSql" />
     <PackageReference Include="Azure.ResourceManager.Redis" />
+    <ProjectReference Include="..\..\..\redisenterprise\Azure.ResourceManager.RedisEnterprise\src\Azure.ResourceManager.RedisEnterprise.csproj" />
+    <!-- <PackageReference Include="Azure.ResourceManager.RedisEnterprise" /> -->
```

Comment:
we are waiting for a new stable version release of redisenterprise.
We will change this project reference to the package reference once that is done.